File: code/1042/1042.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const numbers = lines.filter((l) => !!l).map((l) => parseInt(l));

numbers
  .slice()
  .sort((a, b) => a - b)
  .forEach((n) => console.log(n));
console.log();
numbers.forEach((n) => console.log(n));

File: code/1042/1042.cs
using System;
using System.Linq;

class URI
{
    static void Main(string[] args)
    {
        string[] input = Console
            .ReadLine()
            .Split(' ');
        int[] numbers = input
            .Select(int.Parse)
            .ToArray();
        int[] sortedNumbers = numbers
            .OrderBy(item => item)
            .ToArray();

        PrintArray(sortedNumbers);
        PrintEmptyLine();
        PrintArray(numbers);
    }

    private static void PrintArray(int[] array)
    {
        foreach (int item in array)
        {
            Console.WriteLine(item);
        }
    }

    private static void PrintEmptyLine()
    {
        Console.WriteLine();
    }
}
